packages/repack/src/webpack/plugins/ReactRefreshPlugin.ts

```diff
@@ -1,95 +0,0 @@
-import webpack from 'webpack';
```

Comment:
maybe worth to make it a no-op with a warning "This plugin has been deprecated and is now an no-op. Please switch to `DevelopmentPlugin`..."

Reply:
as mentioned above, I'm not sure anyone used it ever, so we should even skip compatibility layer like that